samples/navigation-fragments/src/test/java/com/instacart/formula/navigation/CounterFragmentFormulaTest.kt

```diff
@@ -0,0 +1,125 @@
+package com.instacart.formula.navigation
+
+import com.google.common.truth.Truth.assertThat
+import com.instacart.formula.test.test
+import kotlinx.coroutines.flow.MutableSharedFlow
+import org.junit.Test
+
+class CounterFragmentFormulaTest {
```

Comment:
It turned out unit tests are not run on CI for `/samples` projects. Adding it here: https://github.com/instacart/formula/pull/460